client/my-sites/themes/controller.js

```diff
@@ -68,6 +69,19 @@ function makeElement( ThemesComponent, Head, store, props ) {
 	);
 };
 
+// Where do we put this? It's client/server-agnostic, so not in client/controller,
+// which requires ReactDom... Maybe in lib/react-helpers?
```

Comment:
Think we should move it over, along with `client-side-effects`, as part of this PR already?